### DISCUSSION TITLE
Github Student developer pack: remote student

### DISCUSSION THREAD

**Discussion by AT-TMC** [community/community#60297] [GitHub Education]
### Hi everyone,

I'm running into an issue of getting my developer pack approved. I currently do not live in the same state as my university, but I attend classes remotely. I cannot even attempt to resubmit another proof of attendance the website won't even let me resubmit.

**Comment by akkefa** [comment]
I am also facing the same issue with the University of colorado MS data science degree. It's a remote degree.

https://github.com/orgs/community/discussions/60024
